### PR TITLE
Fixed Login issues + Upload issues

### PR DESCRIPTION
**Pull request by daniakuli**

### CODE REVIEW

File: Server/routes/upload.js [hunk 1]
```diff
@@ -25,7 +38,7 @@ const upload = multer({ storage: multerStorage });
 
 router.post('/', function (req, res) {
     let upload = req.body;
-    console.log(upload);
+    console.log(upload)
```
Comment (rashel1234) [inline Server/routes/upload.js]: is it necessary?

File: Server/routes/upload.js [hunk 2]
```diff
@@ -11,6 +11,19 @@ import fs from 'fs'
 
 const router = express.Router();
 
+function setDate(nametype){
```
Comment (rashel1234) [inline Server/routes/upload.js]: Please change the name of the function to what it actualy do

File: Server/routes/upload.js (hunk 1 above)
Comment (daniakuli) [reply Server/routes/upload.js]: No i will delete this

File: Server/routes/upload.js (hunk 2 above)
Comment (daniakuli) [reply Server/routes/upload.js]: OK

**Pull request merged**